packages/std/src/errors/backtrace.rs

```diff
@@ -0,0 +1,48 @@
+use core::fmt::{Debug, Display, Formatter, Result};
+use thiserror::Error;
+
+/// This wraps an actual backtrace to achieve two things:
+/// - being able to fill this with a stub implementation in `no_std` environments
+/// - being able to use this in conjunction with [`thiserror::Error`]
+#[derive(Error)]
```

Comment:
Should the backtrace itself really derive `Error`? Isn't it more like a component of an error?

Reply:
Good point. I think I did that while trying to figure out why thiserror complains. It shouldn't be needed